railties/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Allow adding additional authorized hosts in development via RAILS_DEVELOPMENT_HOSTS
+
+    *Josh Abernathy and Debbie Milburn*
```

Comment:
The formatting should be like this:

```suggestion
    *Josh Abernathy*, *Debbie Milburn*
```
See https://github.com/rails/rails/blob/574b519b1aef5963d2b76d95150ace534c4a521c/activerecord/CHANGELOG.md